src/validator.ts

```diff
@@ -439,6 +439,17 @@ export const shouldInitializeGlobalReference = error<Variable>(node =>
   !(node.isAtPackageLevel && node.value.isSynthetic && node.value.is(Literal) && node.value.isNull())
 )
 
+export const shouldInitializeConst = error<Variable>(node => {
+  const isInProgram = getContainer(node)?.kind == 'Program'
```

Comment:
Estamos seguros que queremos usar `getContainer`? Ese buscar en todos los ancestros.
No se puede usar el `is`?

```suggestion
  const isInProgram = getContainer(node)?.is(Program)
```

Reply:
+1 a usar is porque usa el tipo y después entiendo que te queda casteado también